lib/semian/circuit_breaker.rb

```diff
@@ -43,13 +49,16 @@ def mark_failed(error)
 
     def mark_success
       return unless half_open?
-      @successes += 1
+      #@successes += 1
```

Comment:
Commenting code out is usually a bad practice. Any reason why you'd comment it instead of just plain delete it?